README.md

```diff
@@ -53,6 +53,7 @@ ftrrtf_rollbar:
             source_map_enabled: true
             code_version: "some version string, such as a version number or git sha"
             guess_uncaught_frames: true
+            rollbarjs_version: "rollbar_js_version" // i.e. v1.7, if empty then the newest available version will be used
```

Comment:
you should use `#` for comment, and maybe "newest" as a default value? what you think?